### ISSUE TITLE
Assert on moving tab in a split notebook

### ISSUE THREAD

**Issue by Kinaou** [nhold/wxWidgets#122]
As @fastworks said in https://github.com/nhold/wxWidgets/issues/76, it is not possible to move a tab of a split notebook from one area to another:

> Create a pane with a splitter then try to drag one of the two tabs. 
> I get framemanager.cpp(2719) assert failure in layoutadddock flag optionactivenotebook should not be defined to several wxauipaneinfo.

**Comment by mjmacleod** [comment]
hrm, when was this assert added?
Does taking the assert away fix the problem?

**Comment by Kinaou** [comment]
I did that in January (see issue https://github.com/nhold/wxWidgets/pull/80 commit 1dffc8a) and indeed, removing the assert corrects the issue :confused:

``` diff
@@ -2714,8 +2714,10 @@
                             ShowWnd(pane.GetWindow(),true);
                         activenotebookpagefound = true;
                     }
-                    //else
-                        // Add a debug warning
+                    else
+                    {
+                        wxFAIL_MSG("Flag optionActiveNotebook should not be defined to several wxAuiPaneInfo");
+                    }
                 }
                 else
                     {
```

**Comment by mjmacleod** [comment]
Hrm, suspected it might be something like that :)

I think then we can just remove the assert, or do we need to do something else here?

**Comment by Kinaou** [comment]
Reverting (or deleting) this part of code (1dffc8a) seems to be a good option.

**Comment by mjmacleod** [comment]
Above looks fine to me.

**Comment by pkulchenko** [comment]
@Kinaou, are you sending a pull request with this change?

**Comment by pkulchenko** [comment]
@mjmacleod, @nhold, should this issue be closed since #129 has been merged?

**Comment by mjmacleod** [comment]
Hi, yes I believe so.
